CMakeLists.txt

```diff
@@ -96,6 +96,9 @@ list(APPEND OT_PUBLIC_INCLUDES ${PROJECT_SOURCE_DIR}/include)
 if(OT_PLATFORM STREQUAL "posix-host")
     list(APPEND OT_PRIVATE_INCLUDES ${PROJECT_SOURCE_DIR}/src/posix/platform)
     add_subdirectory("${PROJECT_SOURCE_DIR}/src/posix/platform")
+elseif(OT_PLATFORM MATCHES "nrf")
```

Comment:
Since there is multiple chips under 528xx directory, nrf52840 instead?

Reply:
this OT_PLATFORM match also set for the rest nrf platforms like nrf52811